{{ cookiecutter.__project_name_kebab }}/.github/workflows/nightly.yml

```diff
@@ -0,0 +1,30 @@
+name: Nightly Wagtail Test
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+    # At 01:00, daily
+  workflow_dispatch:
```

Comment:
Registering a `workflow_dispatch` trigger here means we can run it on demand. We don't need `trigger-nightly-build.sh`.